config/mongoid.yml

```diff
@@ -1,27 +1,159 @@
 development:
-  sessions:
+  # Configure available database clients. (required)
+  clients:
+    # Defines the default client. (required)
     default:
+      # Defines the name of the default database that Mongoid can connect to.
+      # (required).
+      # database: manuals_publisher_development
```

Comment:
I'm not sure this comment line is correct/relevant is it? Presumably they've been copied from Manuals Publisher...?